index.html

```diff
@@ -151,25 +151,42 @@ <h4>LET’S CONNECT</h4>
         <ul class="myself_ul">
             <li class="language ">
                 <div class="myself_ul_li">
-                <h4>Languages</h4> 
-                <i class="fa-solid fa-angle-down icon"></i>
+                    <h4>Languages</h4>
+                    <i class="fa-solid fa-angle-down icon"></i>
                 </div>
                 <ul>
                     <li><img src="images/js.png" alt=""> JAVASCRIPT</li>
                     <li><img src="images/html.png" alt=""> HTML</li>
                     <li><img src="images/css.png" alt=""> CSS</li>
                 </ul>
             </li>
-            <li class="myself_ul_li"><h4>Framework </h4><i class="fa-solid fa-angle-right icon"></i></li>
+            <li class="myself_ul_li">
+                <h4>Framework </h4><i class="fa-solid fa-angle-right icon"></i>
+            </li>
             <li>
                 <hr>
             </li>
-            <li class="myself_ul_li"> <h4>Skill</h4><i class="fa-solid fa-angle-right icon"></i></li>
+            <li class="myself_ul_li">
+                <h4>Skill</h4><i class="fa-solid fa-angle-right icon"></i>
+            </li>
             <li>
                 <hr>
             </li>
         </ul>
     </section>
+
+    <section class="contact">
+        <h2>Contact me</h2>
+        <p>If you have an application you are interested in developing, a feature that you need built or a project that
+            needs coding. I’d love to help with it </p>
+        <form action="/" method="get">
+            <input type="text" id="fname" name="fname" placeholder="Name" required maxlength="20"  class="form"><br><br>
+            <input type="email" id="email" name="email" placeholder="Email" required class="form"><br><br>
+            <textarea id="w3review" name="w3review" rows="10" cols="50" placeholder="Write your message here"></textarea>
+            <br>
+            <input type="submit" value="Get It Touch" id="submit">
+        </form>
```

Comment:
- [ ] Please know that the form needs to have an `action` and a method `post`, because you need to use [Formspree service](https://formspree.io/html/) to collect the data of the forms. 😁
- [ ] Please know that the name input should be able to accept up to 30 characters instead of 20, kindly change that.
- [ ] Please know that the `textarea` field should be required as well, as the form shouldn't be submitted if one of the fields is empty. 😁